lib/utils/runOpen.js

```diff
@@ -12,7 +12,11 @@ function runOpen(uri, options, log) {
     openMessage += `: ${options.open}`;
   }
 
-  return open(`${uri}${options.openPage || ''}`, openOptions).catch(() => {
+  const pageUrl = /^(https?:)?\/\//.test(options.openPage)
```

Comment:
I think you should use something like https://github.com/ogt/valid-url instead of that simple regex in order to validate if the url is valid.

Reply:
I don't think we need a full separate package to validate the url. I think Regex is enough.

Reply:
I would still prefer to use a better method, but I would left this consideration for others